cypress/support/fragments/eholdings/eHoldingsTitles.js

```diff
@@ -28,9 +28,9 @@ export default {
       eHoldingsTitle.waitLoading(title);
     });
   },
-  create: (packageName, titleName = `title_${getRandomPostfix()}`) => {
-    cy.do(resultSection.find(Button('New')).click());
-    eHoldingsNewCustomTitle.waitLoading();
+  create: (packageName, titleName = `title_${getRandomPostfix()}`, paneTitle) => {
+    cy.do(resultSection.find(Button({ href: '/eholdings/titles/new' })).click());
+    eHoldingsNewCustomTitle.waitLoading(paneTitle);
```

Comment:
This change could affect tests where the method is already used. Please include execution of all tests where the method is used to result report or remove this change.

Reply:
It was one test where this method is used. Added successful run in the description of the pull request